src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  let newState = { ...state };
+  const resultArray = [];
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
```suggestion
  for (const action of actions) {
```

We don't need indexes here so we can use for of loop instead